### ISSUE TITLE
Documentation link is dead

### ISSUE THREAD

**Issue by mgerlach-klick** [tciuro/NanoStore#118]
http://dl.dropbox.com/u/2601212/NanoStore%202.0/html/index.html gives a 404

**Comment by tciuro** [comment]
Yeah. Dropbox must have changed the way links work. When I use their Copy Dropbox Link feature it now shows:

https://www.dropbox.com/s/5b21arjur4unjdi/index.html

The page is renders within Dropbox. I have contacted them about this. Thanks for reporting the issue.

**Comment by tciuro** [comment]
Meanwhile, the documentation can be found here:

https://www.dropbox.com/sh/j9ee5t8y2v963ud/AAAfEnawoYf6Qm5ahrDS11Era?dl=1

The images also seem to be broken. One more reason to host my content on my own server. </sigh>

**Comment by mgerlach-klick** [comment]
Dropbox changed this about a year ago precisely to stop people from using them as a webhost for static pages :)
I also used to do this and have since moved all my stuff over to S3: [http://gerlach.coffee/posts-output/2017-09-17-dropbox-replacement/](http://gerlach.coffee/posts-output/2017-09-17-dropbox-replacement/)

**Comment by mgerlach-klick** [comment]
On second thought, for documentation for a project you may also look into [Github Pages](https://pages.github.com/). This hosts a static page from within a branch of your git repo.
